### ISSUE TITLE
Running the server does not open port

### ISSUE THREAD

**Issue by FlashHUN** [Impostor/Impostor#388]
# Bug Report

## Base Information
- Operating System: Windows 10
- Impostor Version: 1.4.0-ci.173
- Among Us Version: v2021.3.31.3s

## I confirm:
- [x] that I have searched for an existing bug report for this issue.

## Symptoms 
Running the latest (1.4.0-173) version of Impostor Server does not open the port people could join through, even though I have forwarded it on my router and running a different game's server (for example a Minecraft server) opens it.

**Comment by FlashHUN** [comment]
Some additional information: I have my public ip set in the config file (using the default one that's in the zip), and I have also allowed the port through my firewall, so neither of those could be the issue.

**Comment by IrishBruse** [comment]
have you port forwarded your router?

**Comment by FlashHUN** [comment]
> 
> 
> have you port forwarded your router?

"Running the latest (1.4.0-173) version of Impostor Server does not open the port people could join through, **even though I have forwarded it on my router** and running a different game's server (for example a Minecraft server) opens it."

**Comment by IrishBruse** [comment]
22023? you cant use the minecraft port for it

**Comment by FlashHUN** [comment]
> 
> 
> 22023? you cant use the minecraft port for it
> also make sure its udp

I ran the Minecraft server with 22023 as the port, it worked. I forwarded it as TCP-UDP, is that an issue?

**Comment by FlashHUN** [comment]
> 
> 
> > 22023? you cant use the minecraft port for it
> > also make sure its udp
> 
> I ran the Minecraft server with 22023 as the port, it worked. I forwarded it as TCP-UDP, is that an issue?

Can confirm, setting it to UDP only does **not** fix it.

**Comment by IrishBruse** [comment]
how are you checking if its port forwarded online tester dont usually check udp

**Comment by FlashHUN** [comment]
> 
> 
> how are you checking if its port forwarded online tester dont usually check udp

Oh. Didn't know that. I also checked with a friend earlier and he got timed out with a message along the lines of 'reliable packet 7500ms' when trying to join. That's when I looked at what 'canyouseeme.org' says (the port checker I used for about a decade now) and started looking into why it says it's closed instead of trying a couple more times. We'll test it again a bit later, I'll either close the issue if it works or update it if we can't figure it out. Thank you!
